tests/edgeless/shortcut.spec.ts

```diff
@@ -57,13 +57,36 @@ test('shortcut', async ({ page }) => {
   const panButton = locatorEdgelessToolButton(page, 'pan');
   await expect(panButton).toHaveAttribute('active', '');
 
-  await page.keyboard.press('l');
+  await page.keyboard.press('c');
   const connectorButton = locatorEdgelessToolButton(page, 'connector');
   await expect(connectorButton).toHaveAttribute('active', '');
 
+  await page.keyboard.press('l');
+  const lassoButton = locatorEdgelessToolButton(page, 'lasso');
+  await expect(lassoButton).toHaveAttribute('active', '');
+});
+
+test('toggle lasso tool modes', async ({ page }) => {
+  await enterPlaygroundRoom(page);
+  await initEmptyEdgelessState(page);
+  await switchEditorMode(page);
   await page.mouse.click(100, 100);
-  await page.keyboard.press('x');
```

Comment:
This shortcut was removed in https://github.com/toeverything/blocksuite/pull/6573